stl/inc/deque

```diff
@@ -1145,29 +1213,47 @@ public:
 #endif // _ITERATOR_DEBUG_LEVEL == 2
     }
 
-    template <class _Iter, enable_if_t<_Is_iterator_v<_Iter>, int> = 0>
-    void assign(_Iter _First, _Iter _Last) {
+private:
+    template <class _Iter, class _Sent>

```

Comment:
Usually we use `_Se` for the sentinel type and you also did it below. We should settle on one way o write it
```suggestion
    template <class _Iter, class _Se>
```

Reply:
Ranges code uses `_It` and `_Se`. Pre-Ranges code generally uses `_Iter`. Now that we're mixing the two, we have to reconcile conflicting conventions: I picked `_Iter`/`_Sent` as locally consistent with the uses of `_Iter`, but I can see arguments for both `_It`/`_Se` and `_Iter`/`_Se`. I _really_ dislike the latter due to mismatched placeholder lengths, but I could live with either `_Iter`/`_Sent` or `_It`/`_Se`. I'll give other maintainers a chance to register opinions and try to get consensus before making a change here.

Reply:
I am all for `_Iter` / `_Sent`, it was just that both were used here. If you go with `_Iter` / `_Sent` all the way I am perfectly happy

Reply:
Done.